src/resources/action-map/filter.js

```diff
@@ -111,7 +111,7 @@ module.exports = {
   }),
   search: (l, r) => ({
     bool: {
-      filter: (r.match(/(?:[a-zA-Z0-9À-ÖØ-öø-ÿ]|\b[-'’]\b)+/g) || [])
+      filter: (r.match(/(?:[a-zA-Z0-9À-ÖØ-öø-ÿ'’"“”]|\b-\b)+/g) || [])
```

Comment:
This was only allowing the quotes in the middle of string, not "quoted" by them.
In the example provided they were using very rare quotes, I added support for them too...  `“”`

https://regex101.com/r/0DNDcU/1

Reply:
I don't think we want this.

Reply:
The test is valuable to keep though, so I'm not going to close this pr (yet)

Reply:
Rolled back the change in the regex, only keeping the test.
No changes made on service-search rewriter, seems to be working right.